### DISCUSSION TITLE
suggest

### DISCUSSION THREAD

**Discussion by lincanhao** [electerm/electerm#2615] [Ideas]
can we goto sftp fast by ssh address?

**Comment by zxdong262** [comment]
Do you mean when switch to sftp. the address should be same folder as ssh?

**Comment by lincanhao** [comment]
yep!




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "ZHAO ***@***.***&gt;; 
发送时间: 2022年8月23日(星期二) 中午1:44
收件人: ***@***.***&gt;; 
抄送: "@@@@***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [electerm/electerm] suggest (Discussion #2615)





 
Do you mean when switch to sftp. the address should be same folder as ssh?
